ArgoProject/src/systems/DebugSystem.cpp

```diff
@@ -0,0 +1,74 @@
+﻿#include "stdafx.h"
+#include "DebugSystem.h"
+#include "singletons/KeyHandlerSingleton.h"
+#include "singletons/ClientSingleton.h"
+
+app::sys::DebugSystem::DebugSystem(app::sce::SceneType& _targetScene)
+	: m_keyHandler(app::sin::KeyHandler::get())
+	, m_client(app::sin::Client::get())
+	, connected(false)
+	, m_targetScene(_targetScene)
+{
+}
+
+void app::sys::DebugSystem::update(app::time::seconds const & dt)
+{
+	serverTesting();
```

Comment:
```suggestion
	if constexpr (s_DEBUG_MODE)
	{
		serverTesting();
	}
```
Add a `constexpr static bool s_DEBUG_MODE = true;` to the header to allow for easy switching on/off of this system.